config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits-generic.yaml

```
@@ -61,7 +70,15 @@ presubmits:
         - runner.sh
         args:
         - scripts/test-infra/build-image-cross.sh
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
```

Oh, and I think 4gi is too little for this

```suggestion
            memory: 8Gi
```